### DISCUSSION TITLE
Welcome to 2-Minutes-Labs-Solutions Discussions !

### DISCUSSION THREAD

**Discussion by Cloud-Wala-Banda** [Cloud-Wala-Banda/Labs-Solutions#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oyadipe** [comment]
This is very helpful

**Comment by mirekchorazy** [comment]
Hi,
I'm looking for some lab tips https://www.cloudskillsboost.google/focuses/121445?catalog_rank=%7B%22rank%22%3A1%2C%22num_filters%22%3A0%2C%22has_search%22%3Atrue%7D&parent=catalog&search_id=46064065
I managed to do some of it, but the whole thing

**Comment by Amiteshsslv** [comment]
how to use itt bruh
